Bottom: a784e499230f62a1bb944acb2393419f7679fb0a
Top:    b196089660f4a68aef85eaec617ad83790c75f52
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-21 16:53:08 -0700

Refresh of 1.15

---

diff --git a/01_15/Makefile b/01_15/Makefile
new file mode 100644
index 0000000..fd9f01a
--- /dev/null
+++ b/01_15/Makefile
@@ -0,0 +1,5 @@
+main:
+	g++ -o main main.cc
+
+clean:
+	rm main
diff --git a/01_15/main.cc b/01_15/main.cc
new file mode 100644
index 0000000..f8b643a
--- /dev/null
+++ b/01_15/main.cc
@@ -0,0 +1,4 @@
+int main()
+{
+  return 0;
+}
